src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -6397,7 +6397,7 @@ namespace ConvectionCoefficients {
         if (ACH <= 0.5) {
             return Hnatural;
         } else {
-            return Hnatural + ((Hforced - Hnatural) * ((ACH - 0.5) / 0.5));
+            return Hnatural + ((Hforced - Hnatural) * ((ACH - 0.5) / 2.5)); // range for interpolation goes from ACH=0.5 to ACH=3.0 or a range of 2.5
```

Comment:
The equation looks good and the comment is a nice touch to help remind us later.